### PR TITLE
Add the support for video thumbnail

### PR DESCRIPTION
**Pull request by Sunilkumarr**
## Summary
Added the support for video thumbnail upload and set the thumbnail for video 
## Testing
Tested by upload videos and checked the video  thumbnail shows in screen and checked by logout and login and see if the video thumbnail is showing.

### CODE REVIEW

File: Sources/Views/ALKVideoCell.swift [hunk 1]
```diff
@@ -264,37 +265,88 @@ class ALKVideoCell: ALKChatBaseCell<ALKMessageViewModel>,
...
+            photoView.image = fileUtills.getThumbnail(filePath: path)
+        }
+    }
+
+    func loadThumbnail() {
+        guard let message = viewModel, let metadata = message.fileMetaInfo else {
+            return
+        }
+        guard ALApplozicSettings.isS3StorageServiceEnabled() || ALApplozicSettings.isGoogleCloudServiceEnabled() else {
+            photoView.kf.setImage(with: message.thumbnailURL)
```
Comment (mukeshthawani) [inline Sources/Views/ALKVideoCell.swift:300]: What happens if `thumnailURL` is not present? We can set a placeholder.

File: Sources/Views/ALKVideoCell.swift [hunk 2]
```diff
@@ -264,37 +265,88 @@ class ALKVideoCell: ALKChatBaseCell<ALKMessageViewModel>,
...
+            return
+        }
+        guard let thumbnailPath = metadata.thumbnailFilePath else {
+            ALMessageClientService().downloadImageThumbnailUrl(metadata.thumbnailUrl, blobKey: metadata.thumbnailBlobKey) { url, error in
+                guard error == nil,
+                      let url = url
+                else {
+                    print("Error downloading thumbnail url")
```
Comment (mukeshthawani) [inline Sources/Views/ALKVideoCell.swift:308]: Same here, we can set a placeholder.

File: Sources/Views/ALKVideoCell.swift (hunk 1 above)
Comment (Sunilkumarr) [reply Sources/Views/ALKVideoCell.swift]: Added placeholder

File: Sources/Views/ALKVideoCell.swift (hunk 2 above)
Comment (Sunilkumarr) [reply Sources/Views/ALKVideoCell.swift:314]: Added placeholder

**Review by mukeshthawani — APPROVED** [review]

**Pull request merged**